ballerina/Ballerina.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+org = "ballerina"
+name = "observe"
+version = "1.0.1"
```

Comment:
We should add a place holder and replace the version automatically using gradle. Otherwise this will not work with releases. (same for the natives jar version below as well.)

Reply:
Since we are reading from the build-config folder, will that be an issue?